rust/cmsis-pack/src/pdsc/component.rs

```diff
@@ -122,18 +122,19 @@ impl FromElem for ComponentBuilder {
         let class_string = class.clone().unwrap_or_else(|| "Class".into());
         let group_string = group.clone().unwrap_or_else(|| "Group".into());
         let sub_group_string = sub_group.clone().unwrap_or_else(|| "SubGroup".into());
-        let files = get_child_no_ns(e, "files")
-            .map(move |child| {
+        let mut files: Vec<FileRef> = Vec::new();
+        for child in e.children() {
+            if child.tag_name().name() == "files" {
                 log::debug!(
                     "Working on {}::{}::{}::{}",
                     vendor_string,
                     class_string,
                     group_string,
                     sub_group_string,
                 );
-                FileRef::vec_from_children(child.children())
-            })
-            .unwrap_or_default();
+                files = FileRef::vec_from_children(child.children());
```

Comment:
You can break the loop after the assignment I think.

Reply:
Would it make sense to replace the loop with `.find()` then ?

Reply:
Yeah :)